fbchat/_file.py

```diff
@@ -155,6 +155,18 @@ def _from_graphql(cls, data):
             uid=data.get("legacy_attachment_id"),
         )
 
+    @classmethod
+    def _from_list(cls, data):
+        data = data["node"]
+        return cls(
+            width=data["original_dimensions"].get("x"),
+            height=data["original_dimensions"].get("y"),
+            thumbnail_url=data["image"].get("uri"),
+            large_preview=data["image2"],
+            preview=data["image1"],
+            uid=data["id"],
```

Comment:
Use the attachment ID instead of the ID returned by the endpoint.

```suggestion
            uid=data["legacy_attachment_id"],
```